### PR TITLE
Made the menu in the templates dynamic

### PR DESCRIPTION
**Pull request by mstevens83**
Made the menu in the templates dynamic
Also changed "decision tree" wording to "Sapelli project"

### CODE REVIEW

File: geokey_sapelli/views.py [hunk 1]
```diff
@@ -11,14 +11,37 @@
...
+    def get_menu_label():
+        raise NotImplementedError
+    
+    @staticmethod
+    def get_menu_url():
+        raise NotImplementedError
+    
+    def add_menu(self,context):
+        Entry = namedtuple("Entry", ["label", "url", "active"])
+        context['menu_entries'] = [Entry(label=subclass.get_menu_label(), url=subclass.get_menu_url(), active=(self.__class__ == subclass)) for subclass in AbstractSapelliView.__subclasses__()]
```
Comment (mfc-julius) [inline geokey_sapelli/views.py]: >80 :-1:

**Comment by mstevens83** [comment]
KNOW ISSUE: 2 weird tests failures (html output comparisons). Can you have a look Oliver?

**Comment by oliverroick** [comment]
The tests fail because the parameters that are added to the context and passed to the template have changed. 

The tests evaluate if the response from a view equals a reference that is created by rendering the template with the expected context (see [line 231 ff.](https://github.com/ExCiteS/geokey-sapelli/blob/master/geokey_sapelli/tests/test_views.py#L213)). 

In order to fix the tests, you have change the parameters passed into the context when creating the reference.

**Comment by mstevens83** [comment]
Ok, I can try to fix the tests then. But is there an easy way to make django show/log/print the full context passed to a view template?

**Comment by mstevens83** [comment]
All tests pass now.

**Pull request merged**